tip-wlan/resources/environments/dev-amazon-tip.yaml

```diff
@@ -45,47 +40,97 @@ opensync-gw-static:
 opensync-gw-cloud:
+  service:
+    annotations:
+      service.beta.kubernetes.io/aws-load-balancer-type: "nlb-ip"
   enabled: true
   externalhostaddress:
-    ovsdb: opensync-controller.demo.lab.wlan.tip.build
-    mqtt: opensync-mqtt-broker.demo.lab.wlan.tip.build
+    ovsdb: opensync-controller.testcluster.lab.wlan.tip.build
+    mqtt: opensync-mqtt-broker.testcluster.lab.wlan.tip.build
   persistence:
     enabled: false
   filestore:
-    url: "https://wlan-filestore.demo.lab.wlan.tip.build"
+    url: "https://wlan-filestore.testcluster.lab.wlan.tip.build"
 opensync-mqtt-broker:
+  service:
+    annotations:
+      service.beta.kubernetes.io/aws-load-balancer-type: "nlb-ip"
   enabled: true
   replicaCount: 1
   persistence:
     enabled: true
     storageClass: "gp2"
 wlan-cloud-graphql-gw:
   enabled: true
+  ingress:
+    annotations:
+      kubernetes.io/ingress.class: alb
+      alb.ingress.kubernetes.io/scheme: internet-facing
+      alb.ingress.kubernetes.io/group.name: ui
+      alb.ingress.kubernetes.io/certificate-arn: "arn:aws:acm:us-east-1:289708231103:certificate/eeab0cc5-f6d1-4bf2-a125-9dbf10daed42"
+      alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS": 443}]'
+      alb.ingress.kubernetes.io/actions.ssl-redirect: '{"Type": "redirect", "RedirectConfig": { "Protocol": "HTTPS", "Port": "443", "StatusCode": "HTTP_302"}}'
+    enabled: true
+    alb_https_redirect: true
+    hosts:
+    - host: wlan-graphql.testcluster.lab.wlan.tip.build    
+      paths: [
+        /
+        ]
   env:
-    portalsvc: wlan-portal-svc.demo.lab.wlan.tip.build
+    portalsvc: wlan-portal-svc.testcluster.lab.wlan.tip.build
 wlan-cloud-static-portal:
   enabled: true
   env:
-    graphql: https://wlan-graphql.demo.lab.wlan.tip.build
+    graphql: https://wlan-graphql.testcluster.lab.wlan.tip.build
+  service:
+    type: NodePort
   ingress:
+    annotations:
+      kubernetes.io/ingress.class: alb
+      alb.ingress.kubernetes.io/scheme: internet-facing
+      alb.ingress.kubernetes.io/group.name: ui
+      alb.ingress.kubernetes.io/certificate-arn: "arn:aws:acm:us-east-1:289708231103:certificate/eeab0cc5-f6d1-4bf2-a125-9dbf10daed42"
+      alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS": 443}]'
+      alb.ingress.kubernetes.io/actions.ssl-redirect: '{"Type": "redirect", "RedirectConfig": { "Protocol": "HTTPS", "Port": "443", "StatusCode": "HTTP_302"}}'
+    alb_https_redirect: true
     hosts:
-      - host: wlan-ui.demo.lab.wlan.tip.build
+      - host: wlan-ui.testcluster.lab.wlan.tip.build
         paths: [
            /
           ]
 wlan-portal-service:
   enabled: true
   persistence:
     enabled: true
-    storageClass: aws-efs
+    storageClass: gp2
+    accessMode: ReadWriteOnce
     filestoreSize: 10Gi
+  tsp:
+    host: wlan-portal-service.testcluster.lab.wlan.tip.build
+  ingress:
+    enabled: true
+    alb_https_redirect: true
+    tls: []
+    annotations:
+      kubernetes.io/ingress.class: alb
```

Comment:
We might want to use YAML Anchoring to make these annotations more DRY: https://anil.io/blog/symfony/yaml/using-variables-in-yaml-files/
With anchors, we can specify the labels once at the top of the YAML and then just assign them in the 3 or 4 places we need them defined.

Reply:
updated, please take a look